### PR TITLE
I18n build process.

### PR DESCRIPTION
**Pull request by channing-google**
Example usage:
```bash
$ npm run build build-js-de
$ npm run build build-all
```

Also cleaned up the gulpfile, including:
- Delete `buildtools/soy_files.js`, since it should just be a gulp task.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -12,144 +12,284 @@
...
+      .pipe(concatCSS(outFile))
       .pipe(cleanCSS())
-      .pipe(gulp.dest('dist'));
-});
+      .pipe(gulp.dest(DEST_DIR));
+}
+
+// Concatenates and minifies the CSS sources.
+gulp.task('build-css', () => buildCss(false));
+gulp.task('build-css-rtl', () => buildCss(true));
```
Comment (bojeil-google) [inline gulpfile.js]: Clarify: gulp task for building the RTL css.

File: gulpfile.js [hunk 2]
```diff
@@ -12,144 +12,284 @@
...
     port: 4000
   });
 });
 
-
 // Deletes intermediate files.
-gulp.task('clean', () => del(['out/*', 'out']));
+gulp.task('clean', () => fse.remove(TMP_DIR));
 
+// Executes the basic tasks.
```
Comment (bojeil-google) [inline gulpfile.js]: Clarify output only for default language.

File: gulpfile.js [hunk 3]
```diff
@@ -12,144 +12,284 @@
...
+gulp.task('default', ['build-js', 'build-npm', 'build-css', 'build-css-rtl'],
+    () => gulp.start('clean'));
 
-gulp.task(
-    'default',
-    ['build-js', 'build-npm', 'build-css'],
+// Builds everything.
```
Comment (bojeil-google) [inline gulpfile.js]: Clarify: for all localized binaries and css.

**Comment by channing-google** [comment]
Thanks @bojeil-google! PTAL

**Pull request merged**